wallet-new/src/Cardano/Wallet/Kernel/Restore.hs

```diff
@@ -379,54 +381,50 @@ restoreWalletHistoryAsync wallet rootId prefilter progress start (tgtHash, tgtSl
     startingPoint <- case start of
         Nothing -> withNode $ getFirstGenesisBlockHash genesisHash
         Just sh -> nextHistoricalHash sh >>= maybe (throwM $ RestorationSuccessorNotFound sh) pure
-    restore genesisHash startingPoint NoTimingData
+    restore genesisHash startingPoint 1000
   where
     wId :: WalletId
     wId = WalletIdHdRnd rootId
 
     -- Process the restoration of the block with the given 'HeaderHash'.
```

Comment:
This comment would now be out of date right? If I followed correctly, it's restore _starting from_ the block with the given hash.

Reply:
Fixed :+1: